arithmetization/src/main/java/net/consensys/linea/zktracer/module/exp/ExpOperation.java

```diff
@@ -91,7 +91,7 @@ final void traceComputation(int stamp, Trace trace) {
       pComputationMsb
       */
       // tanzb turns to 1 iff trimAcc is nonzero
-      tanzb = pComputationTrimAcc.slice(0, i + 1).toBigInteger().signum() != 0;
+      tanzb = pComputationTrimAcc.slice(0, i + 1).toUnsignedBigInteger().signum() != 0;
```

Comment:
it seems here you really want a signed bigInt as you ask for its sign no ?

Reply:
No, you just want to distinguish between `0` and `!= 0`. The alternative would be `...toUnsignedBigInteger().compareTo(BigInteger.ZERO)` or creating some dedicated method for it.